docs/contribute/testing.adoc

```diff
@@ -0,0 +1,423 @@
+= Writing and running tests for OpenSCAP
+
+This document should help you with writing and running tests for your pull
+requests. It is recommended to add a new test for every new functionality or
```

Comment:
Maybe we could improve wording of this:

It is recommended to add a new test if a new functionality is added or if a code that is not covered by any test is updated.